.travis.yml

```diff
@@ -0,0 +1,59 @@
+sudo: false
+dist: trusty
+
+language: cpp
+
+compiler: clang
+
+git:
+  depth: 2
+  quiet: true
+
+cache:
+  timeout: 1000
+  directories:
+  - $HOME/llvm
+  - $HOME/build
+
+env:
+  global:
+    - TIMEOUT=2700
+
+before_script:
+    - if [[ -d $HOME/llvm/tools/clang ]]; then
```

Comment:
Maybe move this to a separate .sh file? Except for the reference to TRAVIS_BUILD_DIR, this is not specific to travis, or is it? The reference to TRAVIS_BUILD_DIR could be removed.

This would also allow to extract some parts of the script into functions.

The trailing ; could then also be removed.

Reply:
I could pass it as an argument to script